commands/transform.ts

```diff
@@ -55,38 +54,19 @@ export async function transform(codemodName: string, source: string, options: an
 
     sourceSelected = res.path
   }
-
   const transformerPath = join(transformerDirectory, `${codemodSelected}.js`)
 
-  const args: string[] = []
-
-  if (dry) {
-    args.push('--dry')
-  }
-
-  if (print) {
-    args.push('--print')
-  }
-
-  if (verbose) {
-    args.push('--verbose=2')
+  const args: Options = {
+    dry,
+    silent,
+    print,
+    verbose: verbose ? 2 : 0,
+    babel: false,
+    ignorePattern: '**/node_modules/**',
+    extensions: 'cts,mts,ts,js,mjs,cjs',
   }
 
-  args.push('--no-babel')
-  args.push('--ignore-pattern=**/node_modules/**')
-  args.push('--extensions=cts,mts,ts,js,mjs,cjs')
-
-  const files = await getAllFiles(sourceSelected)
-
-  args.push('--transform', transformerPath, ...files.map((file) => file.toString()))
-
-  console.log(`Executing command: ${green('jscodeshift')} ${args.join(' ')}`)
```

Comment:
Why this information is not sufficient information anymore?

Reply:
Now the jscodeshift runner is used directly, not the binary. Do you want me to keep that information?

Reply:
ok, seems to be unnecessary for now